vm/actor/src/builtin/power/state.rs

```diff
@@ -227,3 +224,24 @@ pub struct CronEvent {
 }
 
 impl Cbor for CronEvent {}
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use clock::ChainEpoch;
+
+    #[test]
+    fn epoch_key_test() {
+        let e1: ChainEpoch = 101;
+        let e2: ChainEpoch = 102;
+        let e3: ChainEpoch = 103;
```

Comment:
idk not a big deal, but would be nice to verify the negative value against specs-actors, or even a 0 vector just to be safe